states/oklahoma/parse_pdf.py

```diff
@@ -0,0 +1,109 @@
+from common import cache_request
+from io import BytesIO
+from pdfminer.pdfinterp import PDFResourceManager, PDFPageInterpreter
+from pdfminer.converter import TextConverter
+from pdfminer.layout import LAParams
+from pdfminer.pdfpage import PDFPage
+from io import StringIO
+import os
+from collections import defaultdict
+
+
+def get_text(url):
+  req = cache_request(url, is_binary=True)
+  with BytesIO(req) as fh:
+    output = StringIO()
+    manager = PDFResourceManager()
+    converter = TextConverter(manager, output, laparams=LAParams())
+    interpreter = PDFPageInterpreter(manager, converter)
+
+    for page in PDFPage.get_pages(fh, set()):
+      interpreter.process_page(page)
+    
+    converter.close()
+    text = output.getvalue()
+    output.close()
+    return text
+
+def makes_lines(text):
+  lines = []
+  line = ''
+  for character in text:
+    if character == '\n':
+      lines.append(line)
+      line = ''
+      continue
+    line += character
+
+  # remove header
+  return lines[5:]
+
+
+def group_rows(lines):
+  add_data = False
+  # There is no email address, and fax and phone share a line.
+  sections = ['COUNTY', 'PHONE', 'MAILING ADDRESS']
+
+  reset_index = 40
+  index = 40
+  
+  rows = []
+
+  for line in lines:
+    if 'State of Oklahoma' in line:
+      index = 77
+      reset_index = 37
```

Comment:
Did you mean this to fall through?

Reply:
Yeah this just means that we've hit the next page.